db/seeds.rb

```diff
@@ -10,3 +10,112 @@
                    { name: 'manager' }
                  ])
 Rails.logger.info 'Roles created.'
+
+Rails.logger.info 'Creating Users...'
+client = User.create!(
+  email: 'client@test.com',
+  first_name: 'John',
+  last_name: 'Doe',
+  password: 'password'
+)
+
+User.create!(
+  email: 'employee@test.com',
+  first_name: 'Anna',
+  last_name: 'Smith',
+  password: 'password'
+)
+User.last.update(user_role: UserRole.find_by(name: 'employee'))
```

Comment:
```suggestion
User.create!(
  email: 'employee@test.com',
  first_name: 'Anna',
  last_name: 'Smith',
  password: 'password',
  user_role: UserRole.find_by(name: 'employee')
)
```
?

Reply:
So this was my original version, but then I started to wonder if this wouldn't be a security issue? My idea behind permissions in this app is that user sign ups as a regular user and then is given permission of employee or manager if applicable (so that there is no risk of a situation in which a potential hacker assign himself in the sign up form a role of employee or manager. In the suggested change, wouldn't this be a problem or this is just seeds and I shouldn't worry about it at all? :D

Reply:
It sounds good. Both approaches work for me.